app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,32 @@
  *= require_tree .
  *= require_self
  */
+
+ #post {
+   padding: 3px 3px 3px 3px;
+   border-style: solid;
+   border-width: 2px;
+   border-color: #445561;
+   margin-top: 40px;
+   margin-right: 20%;
+   margin-left: 20%;
+   min-width: 500px;
+ }
+
+ #post-body {
+   background-color: #f5f5f5;
+   padding: 20px 40px 20px 40px;
+ }
+
+ #post-footer {
+   background-color: #818f98;
+   padding: 20px 20px 40px 20px;
+ }
+
+ #col-left {
+   float: left;
+ }
+
+ #col-right {
+   float: right;
+ }
```

Comment:
Prefer classes to IDs for formatting — so `.post-body` etc. That's the convention — but it's also worth bearing in mind that IDs should also be unique in a page.

Also best to leave `application.css` as empty as possible, just a place that other files get included, and to put this in a separate file. Maybe `posts.css` or something. Otherwise it gets massive, hard to maintain, and a big source of merge conflicts.